parts/k8s/manifests/kubernetesmaster-kube-apiserver.yaml

```diff
@@ -48,6 +48,15 @@ spec:
         - "--requestheader-extra-headers-prefix=X-Remote-Extra-"
         - "--requestheader-group-headers=X-Remote-Group"
         - "--requestheader-username-headers=X-Remote-User"
+        - "--anonymous-auth=false"
+        - "--profiling=false"
```

Comment:
how does this affect each existing kubernetes version including those created by acs-engine, and those created by services that use acs-engine?

Reply:
These changes are based on CIS benchmark, for example, "--anonymous-auth=false" is recommended here:
https://workbench.cisecurity.org/sections/35552/recommendations/98632

I think ideally, each version of Kubernetes should have their own security guidelines, which translates into a bunch of recommended argument settings in yaml files.

For now, the CIS benchmark I used is for Kubernetes 1.8, the recommendations looks pretty generic across multiple versions of Kubernetes to me. So I have not see the needs to do some further versioning yet. Plus, these changes will need to pass multiple version e2e tests before they will hit prod.

Yes, it should affect both acs-engine and the RPs that consumes it.